cdc/processor/sinkmanager/manager.go

```diff
@@ -93,7 +94,7 @@ type SinkManager struct {
 		// When every time we want to create a new factory, version will be increased and
 		// errors will be replaced by a new channel. version is used to distinct different
 		// sink factories in table sinks.
-		version uint64
+		version atomic.Uint64
```

Comment:
version is protected by a lock. Seems it's unnecessary to be an atomic.

Reply:
I added comments, pointing out that this lock is only used to protect the f field. I think this can reduce the mental burden of using this lock and prevent similar blocking problems in the future.